models/social/arbitrum/cipher_arbitrum_base_trades.sql

```diff
@@ -0,0 +1,32 @@
+{{ config(
+    schema = 'cipher_arbitrum',
+    tags = ['dunesql'],
+    alias = alias('base_trades')
+    )
```

Comment:
you're missing all the incremental config setup here (and other project level spells)